packages/@lwc/template-compiler/src/codegen/index.ts

```diff
@@ -16,6 +16,7 @@ import {
     isStringLiteral,
     isForBlock,
     isIf,
+    isIfBlock,
```

Comment:
Having both `isIf` and `isIfBlock` is a bit confusing. Are there better names we could use here to distinguish these?

Reply:
We could rename the previous `if` to `ifTrueFalse` or `ifLegacy` or something else, but that's a fairly extensive refactor. If possible, I'd like to do that in a separate PR.